### PR TITLE
Target Staking Ratio

### PR DESCRIPTION
**Pull request by fritzschoff**

### CODE REVIEW

File: containers/Connector/Connector.tsx [hunk 1]
```diff
@@ -41,6 +41,18 @@ import { useMemo } from 'react';
 const useConnector = () => {
 	const [network, setNetwork] = useRecoilState(networkState);
 	const [provider, setProvider] = useState<ethers.providers.Provider | null>(null);
+	const [L1DefaultProvider, _] = useState<ethers.providers.BaseProvider>(
+		loadProvider({
+			infuraId: process.env.NEXT_PUBLIC_INFURA_PROJECT_ID,
+			networkId: NetworkIdByName.mainnet,
+		})
+	);
+	const [L2DefaultProvider, __] = useState<ethers.providers.BaseProvider>(
```
Comment (bachstatter) [inline containers/Connector/Connector.tsx:50]: This doesn't need to me state? Can just be a const?

File: sections/shared/Layout/SideNav/DesktopSideNav.tsx [hunk 2]
```diff
@@ -28,19 +28,39 @@ import SubMenu from './DesktopSubMenu';
...
 	const currencyRateChange = useGetCurrencyRateChange(sevenDaysAgoSeconds, 'SNX');
 	const cryptoBalances = useCryptoBalances(delegateWallet?.address ?? walletAddress);
 	const synthsBalancesQuery = useSynthsBalancesQuery(delegateWallet?.address ?? walletAddress);
 	const isL2 = useRecoilValue(isL2State);
 	const { clearSubMenuConfiguration } = UIContainer.useContainer();
+	const { useSNXData } = useSynthetixQueries();
+	const { L1DefaultProvider } = Connector.useContainer();
+	const lockedSNXQuery = useSNXData(L1DefaultProvider!);
+
+	useEffect(() => {
```
Comment (bachstatter) [inline sections/shared/Layout/SideNav/DesktopSideNav.tsx:49]: Instead of a `useState` + `useEffect` this can just be a `useMemo`

**Review by bachstatter — APPROVED** [review]
Nice! It would be nice to resolve the js-monorepo stuff before going to master.. Other that that looks good

**Comment by fritzschoff** [comment]
> Nice! It would be nice to resolve the js-monorepo stuff before going to master.. Other that that looks good

yes lets wait!

**Pull request merged**